tests/playbooks/test-occm-e2e.yaml

```diff
@@ -12,11 +12,21 @@
   roles:
     - role: install-golang
     - role: install-devstack
+      enable_services:
```

Comment:
does this mean every new job need specify this? since I have an upcoming CI PR..

Reply:
https://github.com/kubernetes/cloud-provider-openstack/commit/624942951770345fa30b065ef43e2f6eeaf440da adds a default definition for the `enable_services` variable.